lib/elixir/lib/enum.ex

```diff
@@ -2435,15 +2434,16 @@ defmodule Enum do
 
   ## uniq
 
-  defp do_uniq([h|t], acc, fun) do
+  defp do_uniq([h|t], set, fun) do
```

Comment:
Not really a deal breaker, but it feels that keeping `acc` will make this patch clearer and help to avoid needless changes if the need for `HashSet` change arises.